packages/typespec-ts/test/modularUnit/scenarios/operations/operations.md

```diff
@@ -928,14 +914,135 @@ export async function _testDeserialize(
 
 export function test(
   context: Client,
-  options: TestOptionalParams = { requestOptions: {} },
+  options: TestOptionalParams = { requestOptions: {} }
 ): PagedAsyncIterableIterator<string> {
   return buildPagedAsyncIterator(
     context,
     () => _testSend(context, options),
     _testDeserialize,
     ["200"],
-    { itemName: "lists", nextLinkName: "nextLink" },
+    { itemName: "lists", nextLinkName: "nextLink" }
+  );
+}
+```
+
+# should serialize name normalize in url template
```

Comment:
can you move this ut into `packages/typespec-ts/test/modularUnit/scenarios/operations/pathParam` this folder? this case is url template mishandled path parameters, maybe we should also add one for query/?